layouts/partials/fragments/hero.html

```diff
@@ -1,9 +1,8 @@
 {{ $hero := .self.Params }}
-{{ $dot := .dot }}
 {{ $name := .self.Name }}
 {{ $name := replace $name "/index.md" "" }}
 {{ $name := strings.TrimSuffix ".md" $name }}
-{{ $path := $dot.File.Path }}
+{{ $path := .self.File.Path }}
```

Comment:
Haha totally missed that as a workable solution.

Reply:
Yeah I was surprised to see $dot in the single layout.